package.json

```diff
@@ -33,7 +33,7 @@
     "lint": "eslint \"src/**/*.ts\" && (cd test && eslint \"**/*.ts\")"
   },
   "dependencies": {
-    "@kubernetes/client-node": "^0.14.2",
+    "@kubernetes/client-node": "^0.14.3",
```

Comment:
0.14.3 removed the use of net-keepalive, so we also can remove dependencies we install in Dockefile
https://github.com/kubernetes-client/javascript/commit/a3e6c0aff670d2441c9b31691f36e3d159ad52ec